src/components/panels/NewsPanel.vue

```diff
@@ -55,7 +55,7 @@ export default {
       window.open(url, '_blank');
     },
     getimgling: function(url) {
-      return 'https://img.mybitti.de/o_webp/' + url;
+      return 'https://cdn.warframestat.us/o_webp/' + url;
```

Comment:
Could we add `,rs_760_fit` while we're here?

Reply:
and maybe respell `getimgling` to `getImgSrc`